app/assets/v2/images/grants/hall_of_fame/hall_of_fame.svg

```diff
@@ -0,0 +1,58 @@
.11 0.378608C1258.67 10.0502 1253.23 21.0927 1250.11 32.8756C1246.99 44.6585 1246.24 56.951 1247.91 69.0511C1249.59 81.1512 1253.65 92.822 1259.86 103.397V103.397ZM1273.35 95.6699C1281.15 108.943 1292.72 119.651 1306.62 126.442C1320.51 133.233 1336.1 135.8 1351.41 133.82C1366.72 131.84 1381.06 125.401 1392.62 115.318C1404.18 105.234 1412.45 91.9589 1416.37 77.1707C1420.28 62.3825 1419.68 46.7456 1414.64 32.2372C1409.59 17.7289 1400.33 5.00082 1388.02 -4.33745C1375.71 -13.6757 1360.91 -19.2048 1345.48 -20.2254C1330.06 -21.2461 1314.71 -17.7125 1301.37 -10.0714C1283.48 0.170989 1270.47 17.065 1265.21 36.8942C1259.95 56.7234 1262.87 77.8634 1273.33 95.6637L1273.35 95.6699ZM1286.81 87.9367C1293.05 98.5548 1302.31 107.122 1313.43 112.554C1324.55 117.987 1337.02 120.041 1349.26 118.457C1361.51 116.873 1372.98 111.722 1382.23 103.655C1391.48 95.5887 1398.09 84.9687 1401.23 73.1382C1404.36 61.3077 1403.88 48.7982 1399.84 37.1916C1395.81 25.5849 1388.4 15.4024 1378.55 7.93161C1368.7 0.460835 1356.86 -3.96264 1344.52 -4.77944C1332.18 -5.59623 1319.9 -2.76967 1309.23 3.34281C1302.14 7.40144 1295.94 12.809 1290.98 19.2568C1286.02 25.7046 1282.4 33.0663 1280.31 40.9216C1278.23 48.7769 1277.74 56.972 1278.85 65.0389C1279.97 73.1057 1282.67 80.8864 1286.81 87.9367ZM1300.3 80.2097C1304.98 88.1813 1311.93 94.6142 1320.27 98.6948C1328.61 102.775 1337.97 104.32 1347.16 103.134C1356.35 101.947 1364.97 98.0828 1371.91 92.0295C1378.85 85.9761 1383.81 78.0057 1386.17 69.1266C1388.52 60.2475 1388.16 50.8586 1385.13 42.1476C1382.1 33.4367 1376.53 25.7951 1369.14 20.1896C1361.75 14.5842 1352.86 11.2666 1343.6 10.6567C1334.34 10.0468 1325.12 12.1719 1317.12 16.7632C1306.39 22.915 1298.59 33.0531 1295.44 44.9499C1292.28 56.8467 1294.03 69.5289 1300.3 80.2097V80.2097ZM1327.28 64.7572C1328.84 67.4118 1331.16 69.5536 1333.94 70.9118C1336.72 72.2699 1339.83 72.7834 1342.9 72.3873C1345.96 71.9911 1348.83 70.7032 1351.14 68.6864C1353.45 66.6695 1355.1 64.0143 1355.89 61.0566C1356.67 58.0988 1356.55 54.9714 1355.54 52.0697C1354.53 49.168 1352.68 46.6225 1350.22 44.755C1347.75 42.8875 1344.79 41.7819 1341.71 41.5781C1338.62 41.3742 1335.55 42.0813 1332.89 43.6098C1331.11 44.624 1329.56 45.9757 1328.32 47.5876C1327.08 49.1994 1326.18 51.0399 1325.66 53.0038C1325.14 54.9677 1325.01 57.0166 1325.29 59.0333C1325.57 61.05 1326.25 62.995 1327.28 64.7572ZM1313.8 72.4842C1316.91 77.7934 1321.55 82.077 1327.1 84.7933C1332.66 87.5096 1338.9 88.5365 1345.02 87.7443C1351.14 86.9521 1356.88 84.3762 1361.51 80.3425C1366.13 76.3088 1369.44 70.9985 1371 65.083C1372.57 59.1675 1372.33 52.9125 1370.31 47.1092C1368.29 41.3058 1364.59 36.2148 1359.66 32.4798C1354.74 28.7448 1348.82 26.5336 1342.65 26.1259C1336.48 25.7182 1330.34 27.1323 1325 30.1894C1321.46 32.2184 1318.36 34.922 1315.88 38.1458C1313.4 41.3696 1311.58 45.0505 1310.54 48.9781C1309.5 52.9056 1309.25 57.0031 1309.81 61.0363C1310.37 65.0695 1311.72 68.9596 1313.8 72.4842V72.4842Z" stroke="#FFDB4C" stroke-width="3" stroke-miterlimit="10"/>
```

Comment:
mind compressing with [svgo ](https://github.com/svg/svgo)?

Reply:
Yes, I have compressed both files in that folder.